README.md

```diff
@@ -39,7 +39,7 @@ it('calls componentDidMount after mounting', () => {
     assert(spy.calledOnce);
 });
 
-it('does something with specific initial state', () => {
+it('does something with specified initial state', () => {
```

Comment:
I mean "specific" and not "specified" in this case. An initial state could be specified in the Component itself (e.g. it's dynamically set) and this would overwrite it.